pwiz_tools/Skyline/Executables/SkylineBatch/SkylineBatch/SkylineBatch.csproj

```diff
@@ -336,6 +337,13 @@
       <HintPath>..\..\..\..\Shared\Lib\Newtonsoft.Json.dll</HintPath>
       <EmbedInteropTypes>False</EmbedInteropTypes>
     </Reference>
+    <Reference Include="PanoramaClient, Version=1.0.0.0, Culture=neutral, processorArchitecture=MSIL">
+      <SpecificVersion>False</SpecificVersion>
+      <HintPath>..\..\..\..\Shared\PanoramaClient\bin\x64\Debug\PanoramaClient.dll</HintPath>

```

Comment:
I think PanoramaClient should be a project in "SkylineBatch.sln", and then "SkylineBatch.csproj" should reference the PanoramaClient project instead of the .dll.
That way, Visual Studio will figure out how to build the .dll and put it in the right place and it will build the correct debug or release variant.

Reply:
I tried adding the PanoramaClient project to the SkylineBatch solution, however PanoramaClient has references to the Common and CommonUtil. How should I reference these projects?

Reply:
It looks like Vagisha is working on removing the dependency between PanoramaClient and Common in PR #2642

Reply:
Ok great! I added the CommonUtil project to the solution as well to resolve that dependency. Should I wait for PR #2642 to be merged before committing this change?

Reply:
Committed a change that added CommonUtil and PanoramaClient to the solution